operations.js

```diff
@@ -506,7 +506,8 @@ Operations.prototype._sweepOps = function _sweepOps(ops, direction, callback) {
 
         var id = opKeys[i];
         var op = ops[id];
-        if (op === undefined) {
+
+        if (op === undefined && (id in ops)) {
```

Comment:
This seems incorrect. This is supposed to test whether the operation was never cataloged, or if it was eventually removed. Please explain.